README.md

```diff
@@ -20,13 +18,11 @@ Hence, we develop OpenPose-Plus, a fast and flexible pose estimation framework t
 - Switchable neural networks (e.g., changing VGG to MobileNet for minimal memory consumption)
 - Integrated training on a single GPU and multiple GPUs
 
-## Work in progress
-
 This project is still under active development, some of the TODOs are as follows:
 - Parallel training (experimental)
 - Pose Proposal Networks, ECCV 2018
 
-## Training your model
+## Custom Model Training
 
 Training the model is implemented using TensorFlow. To run `train.py`, you would need to install packages, shown
 in [requirements.txt](https://github.com/tensorlayer/openpose-plus/blob/master/requirements.txt), in your virtual environment (Python <=3.6):
```

Comment:
could us install using : pip install requirements.txt    ?

Reply:
already show this